tests/cover/test_datetimes.py

```diff
@@ -122,8 +122,8 @@ def test_can_find_before_the_year_2000():
 
 
 def test_can_find_each_month():
-    for i in hrange(1, 12):
-        minimal(dates(), lambda x: x.month == i)
+    for i in hrange(12):
```

Comment:
Good spot! I slightly preferred the old behaviour where we didn’t have an off-by-one inside the loop, but OTOH only getting 11 months. Could we iterate over `hrange(1, 13)` instead?

And separately, while we’re here, maybe change `i` to something more descriptive like `month`?